crates/solidity/testing/snapshots/cst_output/EventDefinition/no_parens/generated/0.4.11-failure.yml

```diff
@@ -0,0 +1,21 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ event MyEvent;                                                                   │ 0..14
+
+Errors: # 1 total
+  - >
+    Error: Expected OpenParen.
+       ╭─[crates/solidity/testing/snapshots/cst_output/EventDefinition/no_parens/input.sol:1:14]
+       │
+     1 │ event MyEvent;
+       │              │ 
+       │              ╰─ Error occurred here.
+    ───╯
+
+Tree:
+  - EventDefinition (Rule): # 0..15 "event MyEvent;\n"
+      - EventKeyword (Token): "event" # 0..5
+      - Identifier (Token): "MyEvent" # 6..13
+      - SKIPPED (Token): "" # 13..13
```

Comment:
Is the empty `SKIPPED` token expected here? Since we are not actually skipping anything, I wonder if we should create this token?

Reply:
This is how the current error recovery mechanism works - sometimes we have to skip over unexpected tokens but sometimes we mean to recover past a place where we expected something, but didn't find it. We use the same SKIPPED tokens for both cases, and to structurally verify if a given CST node is complete and legal (without SKIPPED tokens, which act as "holes" or invalid markers).

Reply:
Thanks for the context. Definitely not a blocker for this.